### ISSUE TITLE
RBO greater one in extrapolated RBO and top-weightedness calculation

### ISSUE THREAD

**Issue by f4lco** [changyaochen/rbo#5]
While the `rbo` of identical lists is 1, the result of `rbo_ext`, as well as the top-weightedness calculation, is technically outside of the `[0, 1]` interval.

```python
In [1]: import rbo

In [2]: import numpy as np

In [3]: rbo.RankingSimilarity(np.arange(100), np.arange(100)).rbo_ext()
Out[3]: 1.0000000000000002

In [4]: np.isclose(1, rbo.RankingSimilarity(np.arange(100), np.arange(100)).rbo_ext())
Out[4]: True

In [5]: rbo.RankingSimilarity(np.arange(100), np.arange(100)).top_weightness()
Out[5]: 1.0000000000000222

In [6]: np.isclose(1, rbo.RankingSimilarity(np.arange(100), np.arange(100)).top_weightness())
Out[6]: True

In [7]: rbo.RankingSimilarity(np.arange(100), np.arange(100)).rbo() # correct behavior of normal rbo
Out[7]: 1.0
```

**Comment by changyaochen** [comment]
@f4lco I agree with you that it might be a rounding issue. Since we know that its value must be bounded between [0.0, 1.0], maybe we can check this before the value is returned?

On a side note, I could not reproduce the error you have. I got both `rbo_ext()` and `top_weightness()` as exact 1.0, for 30 decimal point. My python version is 3.6.7.

**Comment by changyaochen** [comment]
In any case, I've patched a function to bound the return values to [0, 1]. Will submitted a PR for this, and you can review it before I merge it.

**Comment by f4lco** [comment]
Perfect. For future reference, here are my platform details: I'm on OS X 10.14.4, and Python 3.6.3:

    Python 3.6.3 |Anaconda custom (64-bit)| (default, Oct  6 2017, 12:04:38)
    [GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin

**Comment by changyaochen** [comment]
Should be solved with the latest PR (#6 )
